### DISCUSSION TITLE
Path Starts with page permissions

### DISCUSSION THREAD

**Discussion by andymarden** [requarks/wiki#6942] [Help / Questions]
I want to restrict access to a particular user groups to only one page and all its sub-pages

Let's say the page is /example and all the pages linked from that page are at /example/subpage

I create a page rule which says "Path Starts With" and set it to /example

This provides access to /example but not any of the sub-pages whose paths clearly and verifiably starts with /example...

So I tried setting it to Path Matches Regex /example*/

Same result.

What is going on here and how do I achieve what I want? When it says "path" my interpretation is that it actually means path. Is it possible that it is only matching on the page name itself? Do I really have to prefix every sub page with "example".

Anyone got a clue here?

**Comment by fnafox** [comment]
You might have a deny rule which is overriding the allow rule. This is especially likely if the user is part of multiple groups. Review your permissions.
Also your regex doesn't match what you wanted, try: `/example/.*`

**Comment by andymarden** [comment]
It's one rule only.

Ah - that form of regex thanks - will try.

Still don't know why Path Starts With didn't work - did it work for you to open up all sub-pages?
